.golangci.yml

```diff
@@ -9,13 +11,19 @@ run:
 linters:
   fast: false
   enable:
-  - gofmt
```

Comment:
why is `gofmt` removed

Reply:
sgtm, thanks for the clarification, linking the repo is case others are curious https://github.com/mvdan/gofumpt